packages/core/src/common/errors.ts

```diff
@@ -20,6 +20,9 @@ export const CONTEXTMENU_WARN_DECORATOR_NO_METHOD =
 
 export const HOTKEYS_HOTKEY_CHILDREN = `${ns} <Hotkeys> only accepts <Hotkey> children.`;
 
+export const ICON_DEPRECATED_NAMES =
+    `${ns} The "pt-icon-" prefix for iconName is deprecated and will be removed in 2.0.`;
```

Comment:
with the `IconName` type, we can effectively whitelist valid values and provide editor autocomplete, so it's suddenly possible to both enforce _and suggest_ the correct short names. also, the `pt-` prefix appears nowhere else in component code (just hidden in constants), so i see the icon prefix as crufty and unnecessary here. i'll admit the maintenance burden is minimal; supporting the prefixed versions simply adds weight.

i'm fine with removing the deprecation warnings for now, they can always be restored at a later date.